### PR TITLE
# FIX: Vulnerabilities on Nokogiri

### PR DESCRIPTION
**Pull request by anil291987**
Some vulnerabilities are found on Nokogiri .
Nokogiri is update to '>= 1.13.6'.

### CODE REVIEW

**Comment by beninabox** [comment]
+1 would love to see this merged.

File: slather.gemspec [hunk 1]
```diff
@@ -29,7 +29,7 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency 'clamp', '~> 1.3'
   spec.add_dependency 'xcodeproj', '~> 1.21'
-  spec.add_dependency 'nokogiri', '~> 1.13'
+  spec.add_dependency 'nokogiri', '>= 1.13.6'
```
Comment (knox) [inline slather.gemspec:32]:
```suggestion
  spec.add_dependency 'nokogiri', '~> 1.13', '>= 1.13.6'
```

**Review by knox — APPROVED** [review]

**Comment by ksuther** [comment]
Thanks for the PR!

**Comment by knox** [comment]
Thanks for merging! Hopefully we see it released soon.

**Pull request merged**